packages/calypso-e2e/src/lib/blocks/block-flows/form-patterns.ts

```diff
@@ -101,7 +101,7 @@ export class FormPatternsFlow implements BlockFlow {
 			.locator( `[aria-label="${ this.configurationData.patternName }"]` )
 			.getByRole( 'option' )
 			// These patterns can load in quite slowly, messing with animation wait checks, so let's give extra time.
-			.click( { timeout: 20 * 1000 } );
+			.click( { timeout: 30 * 1000 } );
```

Comment:
Noticed this in testing -- boy, sometimes these patterns can really _drag_, especially when the machine is under high CPU load like we run 'em. Giving even more time for extra safety.

Reply:
Yeah, I agree - AI related block tests can be *really* slow, especially in AT.